templates/development.php

```diff
@@ -1,11 +1,13 @@
 <?php
 /* @var $throwable \Throwable */
-/* @var $this \Yiisoft\ErrorHandler\HtmlRenderer */
+/* @var $request \Psr\Http\Message\ServerRequestInterface|null */
+/* @var $this \Yiisoft\ErrorHandler\Renderer\HtmlRenderer */
 
 use Yiisoft\FriendlyException\FriendlyExceptionInterface;
 
-$theme = $_COOKIE['yii-exception-theme'] ?? '';
-
+if ($request && isset($request->getCookieParams()['yii-exception-theme'])) {
```

Comment:
We can't get cookie in such way if the error happens in the config during DI container initialization. I think we still should use `$_COOKIE['yii-exception-theme']` as fallback.